src/daemon/jsonrpc/api/error.rs

```diff
@@ -96,6 +103,8 @@ pub enum ErrorCode {
     INTERNAL_ERROR = 11000,
     /// An error internal to revault_net, generally a transport error
     TRANSPORT_ERROR = 12000,
+    /// The watchtower refused our signatures
+    WT_SIG_NACK = 13_000,
```

Comment:
Because i find it much more readable and i didn't want to increase the diff with unrelated changes by modifying the others (as i did for instance to add `From` to be able to use `?` directly for some errors in #306 but didn't change all occurrences).